src/ol/style/RegularShape.js

```diff
@@ -220,24 +220,25 @@ class RegularShape extends ImageStyle {
 
   /**
    * Get the image icon.
-   * @param {number} pixelRatio Pixel ratio.
+   * @param {number} [pixelRatio=1] Pixel ratio.
    * @return {HTMLCanvasElement} Image or Canvas element.
    * @api
    */
   getImage(pixelRatio) {
-    if (!this.canvas_[pixelRatio || 1]) {
```

Comment:
@ahocevar, Do you know if this function is ever called with an undefined / null / zero pixel ration?
I suspect it is not, so checking for a pixel ratio > 0 is unnecessary here?

Reply:
Correct, this is unnecessary. The check was introduced with #11277, but I don't see a need for it either.